lib/image_science.rb

```diff
@@ -41,10 +124,11 @@ class ImageScience
   # :method: height
 
   ##
-  # Saves the image out to +path+. Changing the file extension will
-  # convert the file type to the appropriate format.
+  # Saves the image to memory and returns its String representation.
+  # Changing the file extension will convert the file type to the
+  # appropriate format.
   #
-  # :method: save
+  # :method: buffer
```

Comment:
This is a confusing description now. It doesn't save (as far as the user is concerned)... just returns the (binary) string representation.